src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -192,9 +193,16 @@
   // cannot write to a temporary file on local disk either in case there is a failure writing to disk
   // Without keeping the entire batch in memory, there would be no way to recover the records that failed to write
 
+  BigQueryEventConsumer(DeltaTargetContext context, Storage storage, BigQuery bigQuery, Bucket bucket,
+    String project, int loadIntervalSeconds, String stagingTablePrefix, boolean requireManualDrops,
+    @Nullable EncryptionConfiguration encryptionConfig, @Nullable Long baseRetryDelay) {
+    this(context, storage, bigQuery, bucket, project, loadIntervalSeconds, stagingTablePrefix, requireManualDrops,
+      encryptionConfig, baseRetryDelay, null);
+  }
```

Comment:
nit: why not add `datasetName` to the existing constructor?

Reply:
done:)